cli/src/config-schema.json

```diff
@@ -123,13 +123,17 @@
                             ],
                             "default": "curved"
                         },
-                        "default_node": {
+                        "commit-node-template": {
```

Comment:
nit: other templates are defined in `[templates]` section, so I think these node templates should be named something like `templates.log_graph_node`.

Reply:
I'm surprised that `templates` isn't in the config schema, is that intentional?

Reply:
Indeed. I believe it's an oversight. The `revsets`/`revset-aliases` pair is documented.

Reply:
Initial issue fixed, leaving config schema as is since existing templates aren't documented currently anyway.